spartan/environments/devnet.env

```
@@ -0,0 +1,50 @@
+CREATE_ETH_DEVNET=false
+GCP_REGION=us-west1-a
+CLUSTER=aztec-gke-private
+NETWORK=devnet
+NAMESPACE=devnet
+ETHEREUM_CHAIN_ID=11155111
+ETHEREUM_RPC_URLS=REPLACE_WITH_GCP_SECRET
+ETHEREUM_CONSENSUS_HOST_URLS=REPLACE_WITH_GCP_SECRET
+ETHEREUM_CONSENSUS_HOST_API_KEYS=REPLACE_WITH_GCP_SECRET
+ETHEREUM_CONSENSUS_HOST_API_KEY_HEADERS=REPLACE_WITH_GCP_SECRET
+FUNDING_PRIVATE_KEY=REPLACE_WITH_GCP_SECRET
+LABS_INFRA_MNEMONIC=REPLACE_WITH_GCP_SECRET
+ROLLUP_DEPLOYMENT_PRIVATE_KEY=REPLACE_WITH_GCP_SECRET
+OTEL_COLLECTOR_ENDPOINT=REPLACE_WITH_GCP_SECRET
+VERIFY_CONTRACTS=false
```

```suggestion
VERIFY_CONTRACTS=true
```
I'd say it's handy to have the contracts verified (assuming it's not an extra burden)